src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const properties = sourceString
+    .split(';')
+    .map(element => element.trim())
+    .filter(element => element !== '');
+
+  return properties.reduce((prev, currentValue) => {
+    const property = currentValue.split(':');
+
+    prev[property[0].trim()] = property[1].trim();
+
+    return { ...prev };
+  }, {});
```

Comment:
Ok, but mutation of incoming function parameter is a bad practice (now we know that this object is used only here, but it's not always so obvious like in this case)